src/wp-includes/class-wp-user-query.php

```diff
@@ -1010,6 +1037,54 @@ protected function parse_orderby( $orderby ) {
 		return $_orderby;
 	}
 
+	/**
+	 * Generate cache key.
+	 *
+	 * @since 6.3.0
+	 *
+	 * @global wpdb $wpdb WordPress database abstraction object.
+	 *
+	 * @param array  $args Query arguments.
+	 * @param string $sql  SQL statement.
+	 *
+	 * @return string Cache key.
```

Comment:
```suggestion
	 * @param string $sql  SQL statement.
	 * @return string Cache key.
```
Don't need extra line break.